config-connector/solutions/iam/kpt/subnet/Kptfile

```diff
@@ -0,0 +1,51 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: subnet
+packageMetadata:
+  shortDescription: Grant an IAM role to a member for accessing a given subnet
+openAPI:
+  definitions:
+    io.k8s.cli.setters.subnet-name:
+      x-k8s-cli:
+        setter:
+          name: subnet-name
+          value: subnet-solution
+          setBy: package-default
+      description: The name of the new subnet.
+    io.k8s.cli.setters.subnet-ip-range:
+      x-k8s-cli:
+        setter:
+          name: subnet-ip-range
+          value: ${SUBNET_IP_RANGE?}
```

Comment:
Is it possible for us to specify a working ip range for the customers?

Reply:
We could. But It just may conflict with the existing IP range and the customer would get an error. It's harder to set a unique ip address range than a resource name.

Reply:
Because we are creating a new network for the solution, there shouldn't be any conflict (please correct me if I'm wrong).
Try the sample ip range "10.0.0.0/9" and it doesn't need to be a setter.

Reply:
Yeah, you are right, there could be no conflict. IP range removed. It's now just a hardcoded value.